pkg/controller/dataimportcron-controller.go

```diff
@@ -898,30 +897,19 @@ func (r *DataImportCronReconciler) cleanup(ctx context.Context, cron types.Names
 }
 
 func (r *DataImportCronReconciler) deleteJobs(ctx context.Context, cron types.NamespacedName) error {
-	deletePropagationBackground := metav1.DeletePropagationBackground
-	deleteOpts := &client.DeleteOptions{PropagationPolicy: &deletePropagationBackground}
-	selector, err := getSelector(map[string]string{common.DataImportCronLabel: getCronJobLabelValue(cron.Namespace, cron.Name)})
+	deleteOpts := client.DeleteOptions{PropagationPolicy: ptr.To[metav1.DeletionPropagation](metav1.DeletePropagationBackground)}
```

Comment:
Brace yourself for a ptr.To conflict in release branches, but this is definitely better